jekyll/_cci2/server/installation/installing-server-behind-a-proxy.adoc

```diff
@@ -0,0 +1,48 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Installing server behind a proxy
+:page-layout: classic-docs
+:page-description: Learn how to install CircleCI server v4.x behind a proxy.
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+Depending on your security requirements, you might want to install CircleCI server behind a proxy. Installing behind a proxy gives you the power to monitor and control access between your installation and the broader Internet.
+
+Configuring a proxy happens during link:/docs/server/installation/phase-2-core-services#l-installing-behind-a-proxy[Phase 2 - Core services].
+
+toc::[]
+
+[#known-limitations]
+== Known limitations
+
+* The CircleCI hostname must be added to the no-proxy list because the following services: `output processor` and `vm-service`. The application and build-agent share a no-proxy list and are assumed to be behind the same firewall and therefore cannot have a proxy between them.
```

Comment:
@circleci/server could I get some help getting this point to be correct? I'm not 100% sure what we need to get across.

Reply:
@rosieyohannan This makes sense to me and I'm not sure how to improve it. The idea behind this is if you use a proxy, you need to configure the jobs to not use the proxy when connecting back to CircleCI.

Reply:
@ryanwohara sorry it's this part "The CircleCI hostname must be added to the no-proxy list because the following services: `output processor` and `vm-service`" ... feels like a word is missing